ethereum/db/ephem_db.py

```diff
@@ -0,0 +1,53 @@
+from ethereum import utils
+from rlp.utils import str_to_bytes
+
+from .base_db import BaseDB
+
+class EphemDB(BaseDB):
+
+    def __init__(self):
+        self.db = {}
+        self.kv = self.db
+
+    def get(self, key):
+        return self.db[key]
+
+    def put(self, key, value):
+        self.db[key] = value
+
+    def delete(self, key):
+        del self.db[key]
+
+    def commit(self):
+        pass
+
+    def _has_key(self, key):
```

Comment:
This method feels like a duplicate of `__contains__`.  Is there a reason to not just merge the two methods since it appears that all of the `__contains__` calls just delegate to it?